website/src/main/tut/docs/rules/DisableSyntax.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+title: Disable
+---
+
+# DisableSyntax
+
+_Since 0.5.4_
+
+This rule reports errors when a "disallowed" syntax is used. We take advantage of the ScalaMeta parser to quickly
+scan the source code. This rules like the other syntaxtic rules does not require any compilation.
+
+Example:
+
+```scala
+MyCode.scala:7: error: [DisableSyntax.xml] xml is disabled.
+  <a>xml</a>
+  ^
+```
+
+## Configuration
+
+By default, this rule does not disable syntax.
+
+It contains the following elements:
+
+* keywords such as: `null, throw, var, return`
+* carriage return aka <CR> (windows uses <CR><LF>)
+* semicolons (`;`)
+* tabs
+* xml literals
+
+To disallow a syntax:
+
+```
+DisableSyntax {
+  keywords = [
```

Comment:
What do you think about using 

```
Disable.keywords = [ ... ]
Disable.noTabs = true
...
```
instead of `DisableSyntax { ... }`. I think it makes it easier to copy-paste a single setting.